2024/100-cecilia-ethical-datascience.md

```diff
@@ -0,0 +1,41 @@
+# Cecilia Dones: Navigating Ethical Data Science: GDPR, EU AI Act & Responsible Practice
+
+## Upcoming Events
+Join our Meetup group for more events!
+https://www.meetup.com/data-umbrella
+
+## Key Links
+- Transcript: https://github.com/data-umbrella/event-transcripts/blob/main/2024/100-cecilia-ethical-datascience.md
```

Comment:
Can you update this link with 101?

Reply:
Hi, Updating the link with 101 will change the name of the file from 100 to 101. I'm changing it to 101 now.

Reply:
Sorry, that can stay 100, to match the file name.

Reply:
Hi, I've reverted back to 100.